apps/browser/src/autofill/content/autofillv2.ts

```diff
@@ -1088,28 +548,33 @@ function fill(document: Document, fillScript: AutofillScript) {
         el.type ? el.type.toLowerCase() : null)
       ) {
         case "checkbox":
-          shouldCheck =
+          shouldCheck = !!(
```

Comment:
```suggestion
          shouldCheck = Boolean(
```

Nitpick, using Boolean typecasting is more verbose but in my opinion more clear than a double bang operator. Not requesting a change here, just voicing an opinion.